tests/test_autoscaling/test_autoscaling.py

```diff
@@ -75,6 +75,17 @@ def test_create_autoscaling_group_within_elb():
         ),
     )
 
+    as_client.put_scheduled_update_group_action(
+        AutoScalingGroupName="schedule-scaling-auto-sclaing-group",
```

Comment:
```suggestion
        AutoScalingGroupName="schedule-scaling-auto-scaling-group",
```
Though, it might be more consistent with other file content to use `tester_group`

Reply:
[7b4db2e](https://github.com/spulec/moto/pull/5289/commits/7b4db2e1b9e930cf7a38f986a5d8e1aed01a4cf7)